### DISCUSSION TITLE
Frigate and cheap wifi cameras?

### DISCUSSION THREAD

**Discussion by outofsight** [blakeblackshear/frigate#3075] [Ask A Question]
I have few very cheap wifi cameras around home that now I would like to try with frigate to the possible extent. 
The area is of course with no tarffic at all under normal conditions.
I made few test and I get some interesting result but with some "issue", for example:
- artifacts in the videos,
- lot of errors in the log.

Among errors:
- camera_01: Unable to read frames from ffmpeg process.
- No frames received from camera_02 in 20 seconds. Exiting ffmpeg...
- fmpeg process crashed unexpectedly for camera_02.
- camera_04: ffmpeg process is not running. exiting capture thread...

In a try to avoid overloading the cheap cameras processor, I enabled using only one 1080p rtsp stream for all the roles.

I'm using default ffmpeg settings, having customized just the hwaccel sections for my i5-8500 GPU.

There is anything I can do (ffmepg settings? timeouts?) to mitigare the issue and/or make frigate/ffmpeg more "tolerant" about what I suppose it's caused by the cameras being unable to supply a consistent rtsp stream?

**Comment by NickM-27** [comment]
This would be better off as a full issue with a config file, logs, etc. A lot of different things can cause issues with cameras. See [this sections of the docs](https://docs.frigate.video/configuration/camera_specific) for some types of cameras and ffmpeg args to use for them.

**Reply by outofsight** [reply]
Ok, I made it 
https://github.com/blakeblackshear/frigate/issues/3076
